### PR TITLE
APPS-2511 Remove "allProjects": "VIEW" from Nextflow Pipeline Applet

### PR DESCRIPTION
**Pull request by r-i-v-a**
+ minor typo fix

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Securit
 
 ## Unreleased
 
+### Removed
```
Comment (wormsik) [inline CHANGELOG.md:9]: can you plase move the section below Fixed to unify with previous entries?

**Review by wormsik — APPROVED** [review]
just a one minor issue, otherwise LGTM 👍

File: CHANGELOG.md (hunk 1 above)
Comment (r-i-v-a) [reply CHANGELOG.md]: done (categories were in that order at the top, though).

**Review by wormsik — APPROVED** [review]

**Review by mhrvol — APPROVED** [review]
LGTM

**Pull request merged**